scripts/components/FlRow2LineButton.ts

```diff
@@ -42,12 +46,24 @@ export default class FlRow2LineButton extends FlRow2LineButtonDesign {
     }
     set bottomLeftLabelText(value: string) {
         this.lblLeft.text = value;
+        this.lblLeft.dispatch({
+            type: 'updateUserStyle',
+            userStyle: {
+                width: this.lblLeft.font.sizeOfString(value, Screen.width).width
```

Comment:
Maybe we should add fallback value. I think it will throw if sizeOfString return undefined value.

```ts
this.lblLeft.font.sizeOfString(value, Screen.width)?.width || 50
```

I wrote 50 for example. You should write an available value. @suleymanbasbug